wrapper.sh

```diff
@@ -328,28 +437,54 @@ if [[ $import == true ]]; then
    #add update lessons to remote
    cd ../${repo}-ja
    git init
-   remotes=`git remote | grep "swc" | wc -l`
+   remotes=`git remote | grep "swc-ja" | wc -l`
    if [[ remotes -le 0 ]]; then
-       git remote add swc https://github.com/$git_user/$repo-ja.git 
+       git remote add swc-ja https://github.com/$remote_user/$repo-ja.git 
     fi
-   git pull swc master
+   git pull swc-ja master
+
+   # remove files provided by template
+   rm -rf bin/boilerplate
```

Comment:
These need to be deleted for the template to work from `carpentries-i18n/carp-theme`